.circleci/config.yml

```diff
@@ -33,12 +33,16 @@ jobs:
               docker login -u "$DOCKER_REGISTRY_USER" -p "$DOCKER_REGISTRY_PASSWORD" quay.io
               docker push "quay.io/weaveworks/flux:$(docker/image-tag)"
             fi
+            if echo "${CIRCLE_TAG}" | grep -Eq "helm-^[0-9]+(\.[0-9]+)*(-[a-z]+)?$"; then
```

Comment:
When we merge into the master, the helm-operator gets the same tag as flux. Is that what we want?

Reply:
Yes, that's fine. Well, it's fine that they are both named for the commit and branch. It's a bit annoying that changes to the operator code result in a new (but actually unchanged) daemon image, and vice versa.

Reply:
:+1:   I remember we spoke about this problem a while ago.